src/utils/metrics.h

```diff
@@ -559,6 +604,7 @@ ENUM_REG(kth_percentile_type::P999)
 ENUM_END(kth_percentile_type)
 
 const std::vector<double> kKthDecimals = {0.5, 0.9, 0.95, 0.99, 0.999};
+const std::vector<std::string> kKthNames = {"p50", "p90", "p95", "p99", "p999"};
```

Comment:
how about make them as a map or something, then we are sure that they are in pairs.